libs/ui/src/ui_strings/keyboard_shortcut_handlers.tsx

```diff
@@ -11,26 +12,47 @@ export function KeyboardShortcutHandlers(): React.ReactNode {
   const currentLanguageCode = useCurrentLanguage();
   const availableLanguages = useAvailableLanguages();
   const { setLanguage } = useLanguageControls();
+  const audioControls = useAudioControls();
 
   React.useEffect(() => {
     function onKeyPress(event: KeyboardEvent) {
-      // Shift+L: Switch to next available display/audio language:
-      if (event.key === 'L') {
-        const currentLanguageIndex = availableLanguages.findIndex(
-          (l) => l === currentLanguageCode
-        );
+      const currentLanguageIndex = availableLanguages.findIndex(
```

Comment:
nit: move L19-22 into the `case 'L'` block to co-locate the related logic

Reply:
[bd0969b](https://github.com/votingworks/vxsuite/pull/4501/commits/bd0969b9f918d10a3701c3d26833c0c1646cdae7)